pilot/ProfileViewController.swift

```diff
@@ -18,5 +18,27 @@ class ProfileViewController: UITableViewController {
         let customHeaderView = Bundle.main.loadNibNamed("CustomTableHeaderView", owner: self, options: nil)?.first as! UIView
         self.tableView.tableHeaderView = customHeaderView
     }
-    
+ 
+    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        if indexPath.section == 0 {
+            if indexPath.row == 0 {
+                
+            }
+            if indexPath.row == 1 {
+                
+            }
+        }
```

Comment:
I usually don't like empty `if` statements but since these will be implemented later it's okay.

Reply:
Yea this will be changed soon